Libraries/LibRegex/RegexMatcher.cpp

```diff
@@ -126,7 +139,9 @@ RegexResult Matcher<Parser>::match(const Vector<RegexStringView> views, Optional
     s_regex_dbg.print_header();
 #endif
 
-    bool continue_search = (input.regex_options & AllFlags::Global) || (input.regex_options & AllFlags::Multiline); // || (input.regex_options & AllFlags::MatchNotBeginOfLine) || (input.regex_options & AllFlags::MatchNotEndOfLine);
+    bool continue_search = input.regex_options.has_flag_set(AllFlags::Global) || input.regex_options.has_flag_set(AllFlags::Multiline); // || (input.regex_options & AllFlags::MatchNotBeginOfLine) || (input.regex_options & AllFlags::MatchNotEndOfLine);
```

Comment:
commented code, mr kling will complain lol

Reply:
There's a whole bunch, I'll deal with them when the js stuff is happily where it should be :P